test/Conversion/FIRRTLToHW/lower-to-hw.mlir

```diff
@@ -1586,3 +1586,16 @@ firrtl.circuit "PortSym" {
     %0 = firrtl.eq %out, %c1_ui5 : (!firrtl.uint<5>, !firrtl.uint<5>) -> !firrtl.uint<1>
   }
 }
+
+// -----
+
+// Check that a zero-width value shifted right produces a zero.
```

Comment:
Not sure, might belong in `zero-width.mlir` ?

Reply:
I went back and forth. I'll move it over there. I'd really like to get the test organization cleaned up as I'm getting confused about where things should go. Perhaps directories for individual passes? Anyway, future activity.

Reply:
:heart: :+1: 

Yes, I think that would be great and am willing to spend time on this "soon" . let's do it! :grin: